schemas/manifest.schema.json

```diff
@@ -0,0 +1,473 @@
+                    "type": "object",
+                    "required": [],
+                    "additionalProperties": false,
+                    "properties": {
+                        "dir": {
+                            "type": "string",
+                            "pattern": "^/.*$"
+                        },
+                        "subdirs": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            }
+                        },
+                        "owner": {
+                            "type": "string",
+                            "pattern": "^[^:]*(:[rwx-]{3}$)?"
+                        },
+                        "group": {
+                            "type": "string",
+                            "pattern": "^[^:]*(:[rwx-]{3}$)?"
+                        }
+                    }
+                },
+                "database": {
+                    "type": "object",
+                    "required": ["type"],
+                    "additionalProperties": false,
+                    "properties": {
+                        "type": {
+                            "type": "string",
+                            "enum": ["mysql", "postgresql"]
+                        }
+                    }
+                },
+                "install_dir": {
+                    "type": "object",
+                    "required": [],
+                    "additionalProperties": false,
+                    "properties": {
+                        "dir": {
+                            "type": "string",
+                            "pattern": "^/.*$"
+                        },
+                        "owner": {
+                            "type": "string",
+                            "pattern": "^[^:]*(:[rwx-]{3}$)?"
+                        },
+                        "group": {
+                            "type": "string",
+                            "pattern": "^[^:]*(:[rwx-]{3}$)?"
+                        }
+                    }
+                },
+                "permissions": {
+                    "type": "object",
+                    "required": [],
+                    "additionalProperties": false,
+                    "patternProperties": {
+                        "^.[a-z][a-z0-9_]*$": {
+                            "type": "object",
+                            "required": ["url"],
+                            "additionalProperties": false,
+                            "properties": {
+                                "url": {
+                                    "type": "string",
+                                    "pattern": "^/.*$"
+                                },
+                                "show_tile": {
+                                    "type": "boolean"
+                                },
+                                "allowed": {
+                                    "type": "string"
```

Comment:
Apparently this can also be a list of allowed groups ... not sure if that's a bug or a feature